backend/src/common/tierlist.ts

```diff
@@ -1,11 +1,34 @@
-import {User} from "@prisma/client";
+import { User } from "@prisma/client";
 import {
     FullFetchedAssignment,
     Tier,
     Tierlist,
     UserTier
 } from "codetierlist-types";
 
+/** @return a two letter hash of the string */
+export const twoLetterHash = (str: string) => {
+    let hash = 0;
+    for (let i = 0; i < str.length; i++) {
+        hash = ((hash << 5) - hash) + str.charCodeAt(i);
+        hash |= 0;
+    }
+    return Math.abs(hash).toString(36).substr(0, 2);
+};
```

Comment:
Im assuming you tested this and it worked

Reply:
yes